schemas/user.yml

```diff
@@ -56,3 +56,10 @@
 
         Useful if `user.id` or `user.name` contain confidential information and
         cannot be used.
+
+    - name: domain
+      level: extended
+      type: keyword
+      example: domainname
```

Comment:
I'm not a fan of this example, as it doesn't add any information, or doesn't jog the user's memory in giving a domain name that people may recognize.

I would either remove the example, or perhaps replace with something else that's commonly seen as an LDAP or AD domain name. Not sure what that would be, though. "corp", "intranet", "acme"?

Reply:
Removed this, I think it should be obvious for users of this field.